templates/index.html

```diff
@@ -0,0 +1,269 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
```

Comment:
This markup should be split up into includes, so it can be shared with error pages.

Reply:
Definitely, this was just copied over from the old GUI template and will be synced with the bases as mentioned above.